render.yaml

```
@@ -0,0 +1,41 @@
+# Describe the services required to run the ARPA reporter on Render
+
+services:
+  # web server
+  - type: web
+    name: arpa-reporter-web
+    env: node
+    region: ohio
+    branch: main
+    buildCommand: yarn install && yarn build
+    startCommand: yarn run prod
+    healthCheckPath: /api/health
+    disk:
+      name: Uploads
+      mountPath: /var/data/uploads
```

```suggestion
      name: DataDir
      mountPath: /var/data
```
i know that's not what we currently have, but i wonder if the entire data dir, not just the uploads sub-path, should be the volume...

---

Yeah, I like this idea. 👍  It seems more future proof.